dashboard/app/serializers/api/v1/section_instructor_info_serializer.rb

```diff
@@ -4,7 +4,9 @@ class Api::V1::SectionInstructorInfoSerializer < ActiveModel::Serializer
   attributes :status, :instructor_name, :instructor_email
 
   def instructor_name
-    object.instructor.name
+    if object.active?
+      return  object.instructor.name
+    end
```

Comment:
You've got an extra space in there.
```suggestion
    if object.active?
      return object.instructor.name
    end
```


FYI a shorter way to do it, though I have no strong opinion on which way is more readable.
```suggestion
    object.instructor.name if object.active?
```